manual/cops_layout.md

```diff
@@ -404,7 +422,7 @@ possible to group categories of macros.
        - attr_writer
 ```
 
-### Example
+### Example bad code
```

Comment:
This shows some problems that have to be addressed in the documentation.

Reply:
Yes. There seems to be a problem point in class comment format of this cop.
I hope to fix these individually.